devel/orc/Portfile

```diff
@@ -31,3 +32,7 @@ platform darwin 8 {
     test.env-append        "DYLD_LIBRARY_PATH=${build_dir}/${name}:${build_dir}/${name}-test"
     destroot.env-append    "DYLD_LIBRARY_PATH=${build_dir}/${name}"
 }
+
+# stop muniversal-1.1 from adding --host & --build
+triplet.add_host    none
+triplet.add_build   none
```

Comment:
These won't be required if https://github.com/macports/macports-ports/pull/16669 gets merged